src/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -66,6 +70,15 @@ export class TransactionDetailsSummaryItem extends React.Component<
     }
   }
 
+  buyerTotalDisplayAmount = order => {
+    switch (order.mode) {
+      case "BUY":
```

Comment:
I wonder if it's worth having these as string constants somewhere else?

Reply:
good point! @ds300 where would be a good place to have these type of consts?

Reply:
I was going to say "it doesn't matter, typescript will tell you if you're putting the wrong string in there so having a const somewhere is not valuable" But then I noticed that in this case `order` has type `any`, which is bad and unnecessary.

I'll make a suggested change thing for this if I can figure out how to do that.

We should really turn on `noImplicitAny` at some point...

Reply:
I wonder, if there's a way to incrementally add `noImplicitAny`

Reply:
👍 with new approach (not passing order to the methods) i think we are good here,

Reply:
This is a sort of devils advocate perspective, but from a team-wide engineering perspective is `noImplicitAny` really necessary? We already get pretty good typing and security and its been nice seeing people "come online" with TypeScript from all different backgrounds. Turning that setting on seems somewhat pedantic (not to use the term in in a derogatory sense, just that it might not necessarily solve any real problems we've faced and seems to appeal to the more advanced TypeScript users in the group). Either way, I think that might be RFC territory.

Reply:
:+1: to an RFC. I totally feel you not wanting to increase friction for not much benefit. IME it's not much friction at all to be forced to add types to all function parameters, and it can be really helpful to prevent errors early and give intellisense more coverage. Obviously I'm biased though 😅